### PR TITLE
Nazanin/asset_index

### PR DESCRIPTION
**Pull request by nazaninreihani**
show an error to user when asset index is empty (restricted countries)

### CODE REVIEW

File: src/templates/app/resources/asset_index.jsx [hunk 1]
```diff
@@ -6,6 +6,9 @@ const AssetIndex = () => (
         <div className='gr-padding-10'>
             <p id='errorMsg' className='error-msg invisible' />
             <div id='asset-index' className='has-tabs gr-parent' />
+            <p className='notice-msg center-text invisible' id='empty-asset-index'>
+                {it.L('Asset index is unavailable at this time.')}
```
Comment (negarn) [inline src/templates/app/resources/asset_index.jsx]: @ashkanx is this text approved?

File: src/javascript/app/pages/resources/asset_index/asset_index.ui.js [hunk 2]
```diff
@@ -29,6 +30,8 @@ const AssetIndexUI = (() => {
     const populateTable = () => {
         if (!active_symbols || !asset_index) return;
 
+        if (!asset_index.length) $('#empty-asset-index').setVisibility(1);
```
Comment (negarn) [inline src/javascript/app/pages/resources/asset_index/asset_index.ui.js]: do we need to return after this?

Comment (nazaninreihani) [reply src/javascript/app/pages/resources/asset_index/asset_index.ui.js]: return here causes the showLoadingImage fire and the loader is visible constantly

Comment (negarn) [reply src/javascript/app/pages/resources/asset_index/asset_index.ui.js]: so we should hide the loader and return, doesn't make sense to do all these actions to get asset index data and build the table when there's no data to show

File: src/templates/app/resources/asset_index.jsx (hunk 1 above)
Comment (ashkanx) [reply src/templates/app/resources/asset_index.jsx]: No. I added a ticket

**Pull request merged**